File: execution_nodes/submissions/test_cases/cpp/tests/Test2.cpp
#include "Seller.hpp"

using namespace std;

int test()
{
    Seller seller(2);
    int score=0;
    if(seller.sell()==1)
    {
      if(seller.getQuantity()==1)
      {
        seller.setQuantity(0);
        if(seller.sell()==0)
        {
          if(seller.getQuantity()==0)
          {
            score=1;
          }
        }
      }
    }
    return score;
}

File: docs/examples/unit_tests/test_cases/python2/tests/Test2.py
from Seller import Seller

class Test:
	def test(self):
		seller = Seller(2)
		score = 0
		if seller.sell()==1:
			if seller.getQuantity()==1:
				seller.setQuantity(0)
				if seller.sell()==0:
					if seller.getQuantity()==0:
						score = 1
						return score
		else:
			return 0

